runtime/stateManager.py

```diff
@@ -271,6 +272,14 @@ def hibike_response_device_values(self, data):
             for key, value in params:
                 self.set_value(value, ["hibike", "devices", uid, key], send=False)
 
+    # pylint: disable=invalid-name
+    def hibike_response_device_disconnect(self, uid):
+        """
+        Delete any history of the device at UID.
+        """
+        devs = self.state["hibike"][1]["devices"]
```

Comment:
To actually access the device dictionary, you want to do self.state["hibike"][0]["devices"][0] to parse through the timestamps.